frame/evm/precompile/contracts/blake2/src/lib.rs

```diff
@@ -39,7 +38,7 @@ impl Precompile for Blake2F {
 		input: &[u8],
 		target_gas: Option<u64>,
 		_context: &Context,
-	) -> core::result::Result<(ExitSucceed, Vec<u8>, u64), ExitError> {
+	) -> core::result::Result<PrecompileOutput, ExitError> {
```

Comment:
how to use this Result in contract. This change will cause some issues with the old usage of precompile contract.

Reply:
https://github.com/rust-blockchain/evm/blob/master/src/executor/stack/mod.rs#L84-L89
```rs
pub struct PrecompileOutput {
	pub exit_status: ExitSucceed,
	pub cost: u64,
	pub output: Vec<u8>,
	pub logs: Vec<Log>,
}
```

The PrecompileOutput contains all fields in the old version, only add `log` field.

Reply:
there is no change for the return value of the Precompile Contract.  So we need nothing to be changed. 
The struct PrecompileOutput will be parsed in the evm inner method. And only the two parameters `exit_status` and `output` will be transparent to our contract, it's the same as the older version.
https://github.com/rust-blockchain/evm/pull/32/files#diff-8b611944560f538201b7edeb4c936bf728d167100a3c97b80f6ba758ac0eb240R534-R552